services/common/src/tests/mocks/dataMocks.tsx

```diff
@@ -1205,7 +1206,16 @@ export const PERMITS: IPermit[] = [
             sub_conditions: [],
             step: "1.",
             display_order: 1,
-            permit_condition_status_code: "NST"
+            permit_condition_status_code: "NST",
+            mineReportPermitRequirement: {
+              report_name: "Test Report",
+              mine_report_permit_requirement_id: 1,
+              cim_or_cpo: "cpo",
+              ministry_recipient: ["MMO"],
+              permit_condition_id: 1639,
+              due_date_period_months: 12,
+              initial_due_date: moment("2024-01-01").toDate(),
```

Comment:
Usually these are the values returned from the server- does it want to be a JS Date object or a string?